### PR TITLE
fix/checkbox

### LINKED ISSUE #239: Replace the fuselage's Checkbox component with our own component
1. Replace the fuselage's Checkbox component with our own component
2. The component should implement our theming system
3. There needs to be a prop to disable interactivity.

### PR DESCRIPTION
**Pull request by KushalSr**
# Fuselage's CheckBox component replaced with own componenet

## Acceptance Criteria fulfillment

- [x] Task 1 - CheckBox Component added.
- [x] Task 2 - Remove fuselage's Checkbox component 
- [x] Task 3

Fixes #239

### CODE REVIEW

File: packages/react/src/components/CheckBox/CheckBox.js
Comment (abhinavkrin) [inline packages/react/src/components/CheckBox/CheckBox.js:1]: Why does checkbox contain MessageBody?

File: yarn-error.log
Comment (abhinavkrin) [inline yarn-error.log:1]: It should not be in the commit

File: package-lock.json
Comment (abhinavkrin) [inline package-lock.json:1]: It seems that you have used npm instead of yarn. We are using yarn workspaces and npm should not be used.

**Review by abhinavkrin — COMMENTED** [review]
Hey, there was no implementation of checkbox.  Please make sure you test your changes before creating PR.

**Comment by KushalSr** [comment]
@abhinavkrin Hey Abhinav, really sorry for the hassale. Thanks for the input, actually I'm very new to open source still-learning.

**Pull request closed without merging**